ViAn/GUI/mainwindow.ui

```diff
@@ -258,9 +258,9 @@
         <item>
          <layout class="QHBoxLayout" name="sliderLayout">
           <item>
-           <widget class="QLabel" name="sliderLabel">
+           <widget class="QLabel" name="currentTimeLabel">
             <property name="text">
-             <string>TextLabel</string>
+             <string/>
```

Comment:
`<string/>`? Is this supposed to be an empty string tag? Shouldn't it be `<string></string>`?